computed-properties-es6/index.html

```diff
@@ -0,0 +1,139 @@
+
+    <title>Computed Property Names (ES6) Sample</title>
+
+    <!-- Add to homescreen for Chrome on Android -->
+    <meta name="mobile-web-app-capable" content="yes">
+    <link rel="icon" sizes="192x192" href="../images/touch/chrome-touch-icon-192x192.png">
+
+    <!-- Add to homescreen for Safari on iOS -->
+    <meta name="apple-mobile-web-app-title" content="Computed Property Names (ES6) Sample">
+
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+    <link rel="apple-touch-icon-precomposed" href="../images/apple-touch-icon-precomposed.png">
+
+    <!-- Tile icon for Win8 (144x144 + tile color) -->
+    <meta name="msapplication-TileImage" content="images/touch/ms-touch-icon-144x144-precomposed.png">
+    <meta name="msapplication-TileColor" content="#3372DF">
+
+    <link rel="icon" href="../images/favicon.ico">
+
+    <link rel="stylesheet" href="../styles/main.css">
+  </head>
+
+  <body>
+    <h1>Computed Property Names (ES6) Sample</h1>
+
+    <!--
+    <p>Available in <a href="http://www.chromestatus.com/feature/PLACEHOLDER">Chrome PLACEHOLDER+</a></p>
+    -->
+    <p>ES6 Computed property names allow you to dynamically create property names using an expression in brackets (<code>[ ]</code>)
+        similar to <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_Accessors">property accessors</a>.
+        Below, computed property names are demonstrated using Object Literals, ES6 Classes and ES6 Symbols.
+    </p>
+    <p>Available in Chrome 42 behind the <code>--harmony-computed-property-names</code> flag</p>
+
+    <div class="output">
+        <pre id="log"></pre>
+    </div>
+
+    <!-- // [START code-block] -->
+    <script>
+        "use strict";
+
+        function log() {
+            document.querySelector('#log').textContent += Array.prototype.join.call(arguments, '') + '\n';
+        }
+
+        // Example 1
+        log('Using Computed property names in an object literal:');
+        var prefix = 'foo';
+        var myObject = {
+                [prefix + 'bar']: 'hello',
+                [prefix + 'baz']: 'world',
+                [prefix + prefix.toUpperCase()]: true
+        };
+
+        log(myObject['foobar']);
+        // -> hello
+        log(myObject['foobaz']);
+        // -> world
+        log(myObject['fooFOO']);
+        // -> true
+
+        // Example 2
+        log('\nUsing Computed method names in an object literal:');
+        var fieldName = "field name";
+        var methodName = "method name";
+        var O = {
+                [fieldName]: "value",
+                [methodName]() { log("A method with a computed name was called"); }
+        };
+        O[methodName]();
+        // -> A method with a computed name was called
+
+        // Example 3
+        log('\nUsing Computed property names with ES6 Classes:');
+        function ID(x) {
```

Comment:
This is not ver useful/illustrative. The V8 unit test has this to ensure that non literals work in that context.